src/ServiceManagement/Compute/Commands.ServiceManagement/Extensions/Diagnostics/NewAzureServiceDiagnosticsExtensionConfig.cs

```diff
@@ -89,7 +89,7 @@ public override string StorageAccountEndpoint
         [Parameter(Position = 6, ValueFromPipelineByPropertyName = true, ParameterSetName = NewExtensionParameterSetName, HelpMessage = "Diagnostics Storage Name")]
         [Parameter(Position = 6, ValueFromPipelineByPropertyName = true, ParameterSetName = NewExtensionUsingThumbprintParameterSetName, HelpMessage = "Diagnostics Storage Name")]
         [ValidateNotNullOrEmpty]
-        public override AzureStorageContext StorageContext
```

Comment:
I will only feel comfprtable if there is an end-to-end test of this.  It is OK if you test this end-to-end by hand for this release, but ifd so, please file an issue for adding or altering test automation to cover thsi scenari (passing a storage context to this cmdlet).